splunk-otel.gemspec

```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "opentelemetry-api", "~> 1.0"
   spec.add_dependency "opentelemetry-exporter-jaeger", ">= 0.20.1", "< 0.23.0"
   spec.add_dependency "opentelemetry-exporter-otlp", ">= 0.21", "< 0.25"
-  spec.add_dependency "opentelemetry-instrumentation-base", "~> 0.21.0"
+  spec.add_dependency "opentelemetry-instrumentation-base", "~> 0.21"
```

Comment:
won't this do more harm than good? this lets consumers accidentally install mismatched packages

Reply:
Also wondering about the trailing `.0`. We have it in the docs for -all.

Reply:
It should provide consumers more flexibility. It is basically the same as the jaeger and otlp dependencies above it, just without an upper limit that has to be bumped later.

Reply:
if you're good I'm good!